source/helpers/path-validator.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-else-return */
+import {ERROR_CODES} from '../constants';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5038858)